package.json

```diff
@@ -108,7 +109,8 @@
     "mocha": "^6.1.4",
     "nyc": "^14.1.1",
     "semantic-release": "^15.13.3",
-    "travis-deploy-once": "^5.0.2"
+    "travis-deploy-once": "^5.0.2",
+    "typeface-roboto": "0.0.75"
```

Comment:
The resolver should not supply a font.

Reply:
it only supplies a font for a test. I can remove the test?